src/main/java/org/testobject/rest/api/model/TestError.java

```diff
@@ -0,0 +1,37 @@
+package org.testobject.rest.api.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class TestError {
+
+	private final String file;
+	private final int lineNumber;
+	private final String error;
+
+	@JsonCreator
+	public TestError(@JsonProperty("file") String file, @JsonProperty("lineNumber") int lineNumber,
+			@JsonProperty("error") String error) {
+		this.file = file;
+		this.lineNumber = lineNumber;
+		this.error = error;
+	}
+
+	public TestError(String error) {
+		this(null, -1, error);
```

Comment:
Let' s change this to have `lineNumber: null`?

Reply:
Will do